### PR TITLE
Translated image page into bahasa indonesia

### PR DESCRIPTION
**Pull request by girikuncoro**
Addressed image page of #15341 

/language id

### CODE REVIEW

File: content/id/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+{{< note >}}
+Gunakan Docker *18.06 ke atas*, versi-versi di bawahnya memiliki _bug_ ataupun tidak mendukung perintah eksperimental. Contohnya https://github.com/docker/cli/issues/1135 yang menyebabkan masalah di bawah containerd.
+{{< /note >}}
+
+Kalo kamu terkena masalah ketika mengunggah _manifest-manifest_ yang rusak, cukup bersihkan _manifest-manifest_ yan glama di `$HOME/.docker/manifests` untuk memulai dari bersih.
+
+Untuk Kubernetes, kami biasanya menggunakan _image-image_ dengan sufiks `-$(ARCH)`. Untuk kompatibilitas (_backward compatibility_), lakukan _generate image-image_ yang lama dengan sufiks. Idenya adalah men-_generate_, misalnya `pause` image yang memiliki _manifest_ untuk semua arsitektur dan misalnya `pause-amd64` yang punya kompatibilitas terhadap konfigurasi-konfigurasi lama atau berkas-berkas YAML yang bisa saja punya _image-image_ bersufiks yang di-_hardcode_.
+
+## Menggunakan Registri Privat (_Private Registry_)
```
Comment (sftim) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
## Menggunakan Registri Privat (_Private Registry_) {#menggunakan-registri-privat}
```

File: content/id/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+- buang `imagePullPolicy` dan gunakan `:latest` _tag_ untuk _image_ yang digunakan.
+- buang `imagePullPolicy` dan juga _tag_ untuk _image_.
+- aktifkan [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) _admission controller_.
+
+Harap diingat kamu sebaiknya hindari penggunaan _tag_ `:latest`, lihat [panduan konfigurasi](/docs/concepts/configuration/overview/#container-images) untuk informasi lebih lanjut.
+
+## Membuat Image Multi-arsitektur dengan Manifest
+
+Docker CLI saat ini mendukung perintah `docker manifest` dengan anak perintah `create`, `annotate`, dan `push`. Perintah-perintah ini dapat digunakan
+untuk membuat (_build_) dan mengunggah (_push_) _manifest_. Kamu dapat menggunakan perintah `docker manifest inspect` untuk membaca _manifest_.
```
Comment (wahyuoi) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
untuk membuat (_build_) dan mengunggah (_push_) manifes. Kamu dapat menggunakan perintah `docker manifest inspect` untuk membaca manifes.
```

there is _manifes_ in kbbi
https://kbbi.kemdikbud.go.id/entri/manifes

File: content/id/docs/concepts/containers/images.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+atau cukup dengan mengatur `DOCKER_CLI_EXPERIMENTAL` variabel _environment_ menjadi `enabled` ketika memanggil perintah-perintah CLI.
+
+{{< note >}}
+Gunakan Docker *18.06 ke atas*, versi-versi di bawahnya memiliki _bug_ ataupun tidak mendukung perintah eksperimental. Contohnya https://github.com/docker/cli/issues/1135 yang menyebabkan masalah di bawah containerd.
+{{< /note >}}
+
+Kalo kamu terkena masalah ketika mengunggah _manifest-manifest_ yang rusak, cukup bersihkan _manifest-manifest_ yan glama di `$HOME/.docker/manifests` untuk memulai dari bersih.
+
+Untuk Kubernetes, kami biasanya menggunakan _image-image_ dengan sufiks `-$(ARCH)`. Untuk kompatibilitas (_backward compatibility_), lakukan _generate image-image_ yang lama dengan sufiks. Idenya adalah men-_generate_, misalnya `pause` image yang memiliki _manifest_ untuk semua arsitektur dan misalnya `pause-amd64` yang punya kompatibilitas terhadap konfigurasi-konfigurasi lama atau berkas-berkas YAML yang bisa saja punya _image-image_ bersufiks yang di-_hardcode_.
```
Comment (wahyuoi) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
Untuk Kubernetes, kami biasanya menggunakan _image-image_ dengan sufiks `-$(ARCH)`. Untuk kompatibilitas (_backward compatibility_), lakukan _generate image-image_ yang lama dengan sufiks. Idenya adalah men-_generate_, misalnya `pause` image yang memiliki manifes untuk semua arsitektur dan misalnya `pause-amd64` yang punya kompatibilitas terhadap konfigurasi-konfigurasi lama atau berkas-berkas YAML yang bisa saja punya _image-image_ bersufiks yang di-_hardcode_.
```

File: content/id/docs/concepts/containers/images.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+https://cs.k8s.io/?q=docker%20manifest%20(create%7Cpush%7Cannotate)&i=nope&files=&repos=
+
+Perintah-perintah ini bergantung pada Docker CLI, dan diimplementasi hanya di sisi CLI. Kamu harus mengubah `$HOME/.docker/config.json` dan mengatur _key_ `experimental` untuk mengaktifkan
+atau cukup dengan mengatur `DOCKER_CLI_EXPERIMENTAL` variabel _environment_ menjadi `enabled` ketika memanggil perintah-perintah CLI.
+
+{{< note >}}
+Gunakan Docker *18.06 ke atas*, versi-versi di bawahnya memiliki _bug_ ataupun tidak mendukung perintah eksperimental. Contohnya https://github.com/docker/cli/issues/1135 yang menyebabkan masalah di bawah containerd.
+{{< /note >}}
+
+Kalo kamu terkena masalah ketika mengunggah _manifest-manifest_ yang rusak, cukup bersihkan _manifest-manifest_ yan glama di `$HOME/.docker/manifests` untuk memulai dari bersih.
```
Comment (wahyuoi) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
Kalau kamu terkena masalah ketika mengunggah manifes-manifes yang rusak, cukup bersihkan manifes-manifes yang lama di `$HOME/.docker/manifests` untuk memulai dari awal.
```

File: content/id/docs/concepts/containers/images.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+Kubernetes memiliki dukungan _native_ untuk [Amazon Elastic Container Registry](https://aws.amazon.com/ecr/), ketika Node adalah
+AWS EC2 _instance_.
+
+Cukup gunakan nama panjang _image_ (misalnya `ACCOUNT.dkr.ecr.REGION.amazonaws.com/imagename:tag`) di dalam definisi Pod.
+
+Semua pengguna klaster yang dapat membuat Pod akan bisa menjalankan Pod yang dapat menggunakan
+_image-image_ di dalam registri ECR.
+
+Kubelet akan mengambil dan secara periodik me-_refresh_ kredensial ECR, yang memerlukan _permission_ sebagai berikut:
```
Comment (wahyuoi) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
Kubelet akan mengambil dan secara periodik memperbarui kredensial ECR, yang memerlukan _permission_ sebagai berikut:
```

super nit, might be ignored

File: content/id/docs/concepts/containers/images.md [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+{{< note >}}
+Cara ini merupakan cara yang direkomendasikan saat ini untuk Google Kubernetes Engine, GCE, dan penyedia cloud lainnya yang
+secara otomatis dapat membuat Node.
+{{< /note >}}
+
+Kubernetes mendukung penentuan _key_ registri pada sebuah Pod.
+
+#### Membuat sebuah Secret dengan Docker Config
+
+Jalankan perintah berikut, ganti dengan nilai huruf besar yang tepat:
```
Comment (wahyuoi) [inline content/id/docs/concepts/containers/images.md]:
```suggestion
Jalankan perintah berikut, ganti nilai huruf besar dengan yang tepat:
```

File: content/id/docs/concepts/containers/images.md (hunk 2 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: ah I just learned this, thanks for pointing out, this means we can also introduce `manifes` to other translated pages

File: content/id/docs/concepts/containers/images.md (hunk 1 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: thanks @sftim for pointing this out, I think this is a good idea, we can introduce this for other pages as well

File: content/id/docs/concepts/containers/images.md (hunk 6 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

File: content/id/docs/concepts/containers/images.md (hunk 5 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

File: content/id/docs/concepts/containers/images.md (hunk 4 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

File: content/id/docs/concepts/containers/images.md (hunk 3 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

File: content/id/docs/concepts/containers/images.md (hunk 2 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

File: content/id/docs/concepts/containers/images.md (hunk 1 above)
Comment (girikuncoro) [reply content/id/docs/concepts/containers/images.md]: fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c

**Comment by girikuncoro** [comment]
@wahyuoi I have addressed all comments, please kindly review once again, thanks!

**Comment by wahyuoi** [comment]
/lgtm
last translation for this milestone :tada: :tada: 
thank you @girikuncoro

**Pull request merged**